src/ComposerConstraint.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\lightning_dev;
+
+class ComposerConstraint {
```

Comment:
This class needs a doc comment, and should be marked `final` to prevent anyone from extending it. It's ours :)